metrics/web/test/base/presentation/widgets/svg_image/widgets/svg_image_test.dart

```diff
@@ -0,0 +1,402 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/widgets/svg_image/strategy/svg_image_strategy.dart';
+import 'package:metrics/base/presentation/widgets/svg_image/widgets/svg_image.dart';
+import 'package:mockito/mockito.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group("SvgImage", () {
+    const testImageSrc = 'src';
+    final strategyMock = _SvgImageStrategyMock();
+
+    testWidgets(
+      "applies the default strategy if the given strategy is null",
```

Comment:
Should we test that the widget is using the given strategy to detect whether the SKIA used?